collector/eventhub/src/test/java/zipkin/collector/eventhub/DummyEventProcessorHost.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.collector.eventhub;
+
+import com.microsoft.azure.eventprocessorhost.IEventProcessorFactory;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+public class DummyEventProcessorHost implements IEventProcessorHost {
+
+  public boolean unrergisterWasCalled = false;
+  public boolean rergisterWasCalled = false;
+
+  @Override
+  public Future<?> registerEventProcessorFactory(IEventProcessorFactory<?> factory) throws Exception {
```

Comment:
Then again, copied the implementation. I agree this is not great or idiomatic in java but just trying to wrap an implementation.